scripts/sem_bleaching_map.py

```diff
@@ -246,33 +292,27 @@ def configure_ccd(self, roi):
         if CCD_ROI:
             # TODO: with andorcam3, translation is possible
             # As translation is not possible, the acquisition region must be
-            # centered. => Compute the minimal centered rectangle that includes the
-            # roi.
+            # centered. => Compute the minimal centered rectangle that includes the roi.
             center = [s / 2 for s in self.ccd.shape[0:2]]
             hwidth = (max(abs(roi[0] - center[0]), abs(roi[2] - center[0])),
                       max(abs(roi[1] - center[1]), abs(roi[3] - center[1])))
-    #        big_roi = (center[0] - hwidth[0], center[1] - hwidth[1],
-    #                   center[0] + hwidth[0], center[1] + hwidth[1])
             res = [int(math.ceil(w * 2)) for w in hwidth]
 
             self.ccd.resolution.value = self.ccd.resolution.clip(res)
 
-            # maximum binning (= resolution = 1 px for the whole image)
-            # CCD resolution is automatically updated to fit
-            self.ccd.binning.value = self.ccd.binning.clip(res)
-
-            # FIXME: it's a bit tricky for the user to have the binning set
-            # automatically, while the exposure time is fixed.
-
-            # TODO: check that the physical area of the final ROI is still within
-            # the original ROI
+            # TODO: check that the physical area of the final ROI is still within the original ROI
         else:
             self.ccd.resolution.value = self.ccd.resolution.range[1]
             logging.info("Using the whole CCD area")
 
         logging.info("CCD res = %s, binning = %s",
-                      self.ccd.resolution.value,
-                      self.ccd.binning.value)
+                     self.ccd.resolution.value,
+                     self.ccd.binning.value)
+
+        # turn on cooling of the scmos
+        self.ccd.fanSpeed.value = 1
+        self.ccd.targetTemperature = 0
```

Comment:
This target temperature is only correct for air-cooled Zylas. The proper way would be to store the fanSpeed and targetTemperature at the beginning (or just before turning the fan off), and use it as the "CCD acquisition" values. Also, instead of setting them up here, they should be immediately restored after the survey acquisition (at the end of `acquire_sem_survey()`).